packages/security/carbon.yml

```diff
@@ -0,0 +1,313 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+library:
+  id: ibm-security
+  name: IBM Security
+  description: Legacy and non-curated design implementations used in security
+  externalDocsUrl: https://pages.github.ibm.com/cdai-design/pal
+assets:
+  decorator:
+      name: Decorator
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/decorator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/story/security-components-decorator
+      tags:
+        - content-element
+  delimited-list:
+      name: Delimited list
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/delimited-list.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/story/security-components-delimitedlist
+      tags:
+        - content-element
+        - data-display
+  external-link:
+      name: External link
+      status: stable
+      type: component
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/external-link.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/story/security-components-externallink
+      tags:
+        - content-element
+        - contextual-nav
```

Comment:
```suggestion
        - contextual-navigation
```
getting this error: 
<img width="1459" alt="image" src="https://user-images.githubusercontent.com/40550942/175609381-be9509da-52b0-474d-8e26-314e56e3e5d6.png">